### ISSUE TITLE
Imports not working for C in the case of a FunctionCall

### ISSUE THREAD

**Issue by nhamidn** [pyccel/pyccel#430]
Imports not working for C when calling a function from the main program

**To Reproduce**
```
from pyccel.decorators import types

@types('int,int')
def rec(b,y):
    return b + y

z = rec(2,4)

```

the generated Program in C 
```
#include "test"
#include "stdlib.h"
int main()
{
    long z;
    z = rec(2,4);
    return 0;
} 
```
Which fail in the compilation stage

```
'test' file not found
#include "test"
```
the import of the function should have the .c extension 
`#include "test.c"`

**Language**
C

**Comment by EmilyBourne** [comment]
> the import of the function should have the .c extension
> #include "test.c"

While this solution will fix the problem it is bad practice to include a .c file as the order of the imports becomes important. We should be importing a .h file but currently this is not generated

**Comment by EmilyBourne** [comment]
This issue should be fixed after issue #431

**Comment by EmilyBourne** [comment]
In order to activate the import tests we first need to implement the print function

**Comment by EmilyBourne** [comment]
This should be fixed after issue #432

**Comment by EmilyBourne** [comment]
Until this is fixed we cannot call functions from a program
